build.gradle

```diff
@@ -201,22 +231,20 @@ tasks.register('SetConnectionTypeIP') {
 }
 
 compileJava.finalizedBy checkAkitInstall
-build.dependsOn installLocalGitHookPostCheckout, installLocalGitHookPreCommit
 discoverroborio.finalizedBy SetConnectionTypeIP
 
 
 // Spotless formatting
-project.build.dependsOn(spotlessApply)
 spotless {
     java {
         target fileTree(".") {
             include "src/main/java/frc/**/*.java"
             exclude '**/build/**', '**/build-*/**'
         }
-        indentWithSpaces(4)
         removeUnusedImports()
         trimTrailingWhitespace()
-        endWithNewline()
+        eclipse().configFile("codestyle.xml")
+        formatAnnotations()
     }
     groovyGradle {
```

Comment:
maybe remove gradle mis and json formats

Reply:
It seems like a good idea to me to leave them, I removed the forced indent for misc tho

Reply:
so you don't prefer the indent with tabs for them?